tests/20-SideMenu/BoardsMenuSection.test.tsx

```diff
@@ -26,62 +27,77 @@ describe("Regular", () => {
   test("Renders section with board menu items", async () => {
     render(<Regular />);
   
-    //TODO: fill this
+    expect(screen.getByText(Regular!.args!.title!)).toBeVisible;
+
+    const boards = screen.getAllByRole("link");
+    expect(boards).toHaveLength((Regular.args as BoardsSectionProps).boards!.length);
+    boards.forEach((board, i) => {
+      expect(board).toHaveTextContent(`!${(Regular.args as BoardsSectionProps).boards[i].slug}`);
+    });
   });
   
-  //Not sure if this should be it's own test or part of the previous test
   test("Board menu items link to boards", async () => {
     render(<Regular />);
-  
-    //TODO: fill this
+    const actionReturn = jest.fn();
+    mocked(action).mockReturnValue(actionReturn);
+    
+    const boards = screen.getAllByRole("link");
+    for (const board of boards) {
+      fireEvent.click(board);
+      await waitFor(() => {
+        expect(action).toBeCalledWith("#slug");
+      });
+    };
   });
   
   test("Correctly renders board without updates", async () => {
     render(<Regular />);
-
-    //Not sure .toBe is the right thing to expect here, but can't try it to figure it out because Testing Library can't find my aria labels
-    expect(screen.getByText("!oncie-den")).toBe(screen.getByRole("link", { name: "oncie-den"}));
+    
+    expect(screen.getByRole("link", { name: "oncie-den"})).toHaveTextContent("!oncie-den");
   });
   
   test("Correctly marks boards with updates", async () => {
     render(<Regular />);
   
-    expect(screen.getByLabelText("gore has new updates", {exact:false}));
+    expect(screen.getByLabelText("gore has new updates")).toHaveTextContent("!gore");
   });
   
   test("Correctly marks outdated boards with updates", async () => {
     render(<Regular />);
   
-    //TODO: fill this
+    expect(screen.getByLabelText("crack has updates")).toHaveTextContent("!crack");
   });
   
   test("Correctly marks current board", async () => {
     render(<Regular />);
   
-    //TODO: fill this
+    expect(screen.getByRole("link", { current: "page" })).toHaveTextContent("!kink-memes");
   });
 
   test("Correctly marks muted board", async () => {
     render(<Regular />);
   
-    //TODO: fill this
+    expect(screen.getByLabelText("anime muted")).toHaveTextContent("!anime");
   });
 });
 
 describe("Empty", () => {
   test("Renders empty section", async () => {
     render(<Empty />);
     
-    // This gives error "Property 'emptyDescription' does not exist on type 'Partial<Partial<BoardsMenuSectionProps>>'." and I don't understand why???
-    // expect(screen.getByText(Empty.args!.emptyDescription))
+    // This might be overkill?
```

Comment:
No, it's good. Usually the rule of thumb I use is "would I be like _ugh, gotta fix this and release an update_" if this were to break? In this case, yes.